eslint.json

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><checkstyle version="4.3"></checkstyle>
```

Comment:
this file should not be added here

Reply:
Hi @zbynek Sir, thanks for the review! I understand that the eslint.json file should not be added here. I have removed it. Apologies for the oversight, and I appreciate the clarification! Please let me know if there are any other changes needed.

Reply:
There are definitely more changes needed. Currently the PR just adds duplicates of existing data in a different format, I'm pretty sure that's not the intention of the original issue. As Kris suggested, it might be useful to look into Decap CMS and the map widget to see if adding new map entries can be simplified, and if GeoJSON is needed for that.

Reply:
Hi @zbynek Sir,

Thank you for the feedback!  I'll explore the Decap CMS documentation and the map widget to see how adding new map entries can be simplified. If GeoJSON integration is essential, I'll refine the implementation accordingly.

As @krisstern Sir mentioned, issue #1 should be addressed first for this PR to be considered. I'd be happy to look into solving it. I have gone through it too. Could you please clarify the expectations for that issue?

Looking forward to your guidance!

Reply:
Hi @biru-codeastromer, you may have better luck studying this repo to see how issue #1 would fit into the picture than asking individual maintainers/members about how you could better carry out the task. Also, it may not be such a good idea to ping individual(s) in your issues/pull requests, and they may or may not be directly involved in the particular isue concerned, and this practice may even be considered rude by some.

Reply:
Hi @krisstern Sir,

I sincerely apologize if tagging maintainers in my PR or issue comments came across as inappropriate—that wasn’t my intention. I’ll ensure to avoid this in the future and respect the collaborative process more carefully.

I’ll dedicate more time to studying the repository to understand how issue #1 integrates into the overall context and identify how I can contribute more effectively. Thank you again for your valuable advice, and I’m truly sorry for this oversight. It won’t happen again in the future.

Reply:
@biru-codeastromer, Zbynek has added a comment to issue #1 which may help you to put things in context better.

Reply:
@krisstern Yes Sir, Looking into it.